### PR TITLE
Add coala_utils>=0.4.7 as requirement

### PR DESCRIPTION
**Pull request by Redridge**
Fixes https://github.com/coala-analyzer/coala/issues/2425

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ setuptools>=19.2
 # Do *not* use 0.3 which is not backwards compatible to common clang versions
 libclang-py3==0.2
 appdirs==1.4.*
-coala_decorators>=0.2.1,==0.2.*
+coala_utils>=0.4.7
```
Comment (sils1297) [inline requirements.txt]: please use `~=`, we should switch back to that anyway. The `>=...,==` combination expresses the same btw. but it doesn't have the benefit I hoped so let's use `~=`

**Comment by sils1297** [comment]
unack 97779e1

**Comment by sils1297** [comment]
otherwise good looking, it's time we merge thi

**Comment by Redridge** [comment]
Btw what does the `~=` operator do?

**Comment by sils1297** [comment]
Compatibility

**Comment by Adrianzatreanu** [comment]
ack 7cda7e3

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**